Client/Application/AppDelegate+PushNotifications.swift

```diff
@@ -73,17 +74,20 @@ extension AppDelegate {
     }
 
     private func openURLsInNewTabs(_ notification: UNNotification) {
+        var sentUrlsToQueue: [URL] = []
```

Comment:
@nishant2718 is this for actually sending urls or the urls we received? 
If its the latter then we should make it `receivedUrlQueue`

Reply:
It's for received - from other devices using "Send Tabs." But yeah it makes more sense as receivedUrlQueue